### ISSUE TITLE
ExtensionError: REST API is not yet supported on POST https://graph.microsoft.com/v1.0/subscriptions

### ISSUE THREAD

**Issue by dkoleary88** [microsoftgraph/microsoft-graph-docs#6869]
There's lacking documentation on the types of possible errors one can receive for many of the endpoints.

In particular, I'm seeing this undocumented error when I try to create a webhook subscription.
POST https://graph.microsoft.com/v1.0/subscriptions

"error": {
  "code": "ExtensionError",
  "innerError": {
    "date": "2020-01-21T09:11:26",
    "request-id": "ad5b74f1-04d6-46bc-8a11-e6617a974e37"
  },
  "message": "Operation: Create; Exception: [Status Code: NotFound; Reason: REST API is not yet supported for this mailbox.]"
}

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2f20de46-dc28-3cde-642a-4a7fb7b2f454
* Version Independent ID: be834d56-fb0c-0f16-31f8-287221ba6cc4
* Content: [Create subscription - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/subscription-post-subscriptions?view=graph-rest-1.0&tabs=http#feedback)
* Content Source: [api-reference/v1.0/api/subscription-post-subscriptions.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/api/subscription-post-subscriptions.md)
* Product: ****
* Technology: **microsoft-graph**
* GitHub Login: @baywet
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by baywet** [comment]
Thanks for your feedback!
What was the payload of your request?
Is this tenant federated with exchange on prem? And that mailbox on prem?

**Comment by dkoleary88** [comment]
The payload was 
```json
{"changeType":"created,updated,deleted","notificationUrl":"https://integrations.float.com/webhook/office365_calendar","resource":"/me/calendars/AAMkAGU4MTY5ZGUyLTVlMzAtNDFhOS04MGE3LTYwNjg1ZjVmZTJmMgBGAAAAAADoFG_RdVaET6kgmoGWqOScBwBOARiQtNb8RbsgwMd_cimLAAAAYzWAAABOARiQtNb8RbsgwMd_cimLAAAAY2FUAAA=/events?startDateTime=2019-11-19T21:36:23.441Z&endDateTime=2023-01-19T21:36:23.449Z","expirationDateTime":"2020-01-24T07:40:25.234Z","clientState":"eyJhbGciOiJIUzI1NiJ9.eyJleHRDYWxlbmRhcklkIjoyMjg3fQ.yZ9s6ySOHvw7ucGzmEGtwDW4Sq9BMHdf7JuLK09Ig1c"}
```
We are still working out if the tenant is federated with exchange on-prem.

**Comment by baywet** [comment]
Thanks! Let me know when you have the details about the mailbox.

**Comment by baywet** [comment]
Assigning to you @svpsiva as it seems to be related to Excahange APIs

**Comment by baywet** [comment]
Hey @dkoleary88
Thank you for your patience on the matter.
This could be caused either:
- because the mailbox is on prem, and the federated server doesn't support forwarding MS Graph Requests.
- because the mailbox has an application policy preventing the app from accessing the mailbox.

On the latter, we recently fixed an issue where the policies would not be evaluated properly in the case of subscriptions requests.
I also authored a PR #9406 (don't hesitate to leave feedback there) to document that error code. The PR will close this issue once it gets merged.
